### ISSUE TITLE
GUI 0.13.0.3 crashes on launch on macOS due to missing library

### ISSUE THREAD

**Issue by rex4539** [monero-project/monero-gui#1682]
macOS 10.13.6 (17G3020)

```
Termination Reason:    DYLD, [0x1] Library missing

Application Specific Information:
dyld: launch, loading dependent libraries

Dyld Error Message:
  Library not loaded: /usr/local/Cellar/openssl/1.0.2p/lib/libcrypto.1.0.0.dylib
  Referenced from: /Applications/monero-wallet-gui.app/Contents/Frameworks/libssl.1.0.0.dylib
  Reason: image not found
```

[monero-wallet-gui_2018-10-18-214832_MacBook-Pro-2011.log](https://github.com/monero-project/monero-gui/files/2493231/monero-wallet-gui_2018-10-18-214832_MacBook-Pro-2011.log)

**Comment by rex4539** [comment]
Looks like the path is hardcoded.

**Comment by skftn** [comment]
Thanks for reporting :)

Was indeed hardcoded due to misconfiguration in the OSX build template and a fix will be shipped in the `.4` release that is due soon.

**Comment by dEBRUYNE-1** [comment]
Duplicate of #1658. This particular issue will be fixed in GUI v0.13.0.4. In addition, a temporary workaround can be found here:

https://monero.stackexchange.com/questions/10364/gui-v0-13-0-3-does-not-start-on-mac-os-x-monero-wallet-gui-cannot-be-opened-bec

**Comment by dEBRUYNE-1** [comment]
+resolved
